src/Symfony/Bundle/FrameworkBundle/Command/SecretsInitCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DependencyInjection\Secrets\JweHandler;
+use Symfony\Component\DependencyInjection\Exception\SecretsOverwriteRequiredException;
+
+final class SecretsInitCommand extends Command
+{
+    protected static $defaultName = 'secrets:init';
+    private $io;
+    private $secretsHandler;
+
+    public function __construct(JweHandler $secretsHandler)
+    {
+        $this->secretsHandler = $secretsHandler;
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Set up encrypted_secrets public/private key pair and encrypted secrets file')
+            ->setHelp(<<<'HELP'
+The <info>%command.name%</info> command creates an empty secrets file based on the location configured encrypted_secrets.secrets_file,
+and generates a key pair based on the values set in encrypted_secrets.private_key_file and encrypted_secrets.public_key_file.
+
+After running this command, use
+
+  <info>php bin/console secrets:add</info> or <info>php bin/console secrets:edit</info> to modify your secrets.
+
+Always store your private key in a secure location outside of version control; you will not be able to recover your secrets without it.
+HELP
+            )
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            $this->secretsHandler->initSecretsFiles($overwriteExisting = false);
+            $this->io->success('Secrets files have been initialized.');
+        } catch (SecretsOverwriteRequiredException $e) {
+            $permissionToOverwrite = $this->io->ask(
```

Comment:
There's a `confirm()` method I believe that handles all the `y/n` stuff for you and returns a clean bool.

Reply:
perfect, thank you! added